src/content/add-to-app/ios/add-flutter-screen.md

```diff
@@ -64,11 +66,12 @@ class FlutterDependencies: ObservableObject {
 
 @main
 struct MyApp: App {
-    // flutterDependencies will be injected using EnvironmentObject.
-    @StateObject var flutterDependencies = FlutterDependencies()
+    // flutterDependencies will be injected through the view environment.
+    @State var flutterDependencies = FlutterDependencies()
```

Comment:
We seem to be inconsistent on this page whether we add a new line between a `struct` declaration and its first member. Could you do a pass on this page and double-check everything matches the recommended Swift style guide? (It's possible I just don't understand the Swift style, if so please ignore this feedback!)